index.js

```diff
@@ -321,3 +322,27 @@ function toHexString(v) {
   lsn = (v >> 0) & 0x0f;
   return m_hex[msn] + m_hex[lsn];
 };
+
+function radsToDeg(radians) {
+  return radians * 180 / Math.PI
+}
+
+function padd(n, p, c)
+{
+  var pad_char = typeof c !== 'undefined' ? c : '0';
+  var pad = new Array(1 + p).join(pad_char);
+  return (pad + n).slice(-pad.length);
+}
+
+function conv_latlon(val)
+{
+  if ( val < 0 )
+  {
+    val *= -1;
+  }
+	
+  var lati = Math.floor(val)
+  var latm = val - lati 
```

Comment:
To me `val % 1` would read easier.

I would also use longer variable names to make it obvious what is being done.